### ISSUE TITLE
Can't save changes after using /sellprice set

### ISSUE THREAD

**Issue by CharlyZA** [ForgeEssentials/ForgeEssentials#2054]
```
[10:45:32] [Server thread/ERROR]: Couldn't process command: 'sellprice save'
java.lang.NullPointerException
        at java.util.TreeMap.getEntry(Unknown Source) ~[?:1.8.0_73]
        at java.util.TreeMap.get(Unknown Source) ~[?:1.8.0_73]
        at com.forgeessentials.economy.commands.CommandSellprice.calcPriceList(CommandSellprice.java:222) ~[CommandSellprice.class:?]
        at com.forgeessentials.economy.commands.CommandSellprice.parse(CommandSellprice.java:107) ~[CommandSellprice.class:?]
        at com.forgeessentials.core.commands.ParserCommandBase.func_71515_b(ParserCommandBase.java:20) ~[ParserCommandBase.class:?]
        at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:94) [z.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:739) [nh.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:718) [nh.class:?]
        at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:37) [ir.class:?]
        at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:9) [ir.class:?]
        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) [ej.class:?]
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) [nc.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
```

**Comment by MickScandalous** [comment]
Im also experiencing this issue

**Comment by spacebuilder2020** [comment]
This is related to issue #2056 

Try removing fast craft and see if you have any difference.

**Comment by CharlyZA** [comment]
Fast craft is only installed on the client side.

**Comment by spacebuilder2020** [comment]
ok

**Comment by spacebuilder2020** [comment]
@CharlyZA Did this issue ever resolve itself?

**Comment by Zethalion** [comment]
Closing this as no response is given

**Comment by MickScandalous** [comment]
There was but I think ya boi removed it

**Comment by Zethalion** [comment]
Tested on 1.12.2
Not yet resolved, reopening
`[17:43:03] [Server thread/INFO] [forgeessentials]: Player "Zethalion" used command "/sellprice "
[17:43:03] [Server thread/WARN] [minecraft/CommandHandler]: Couldn't process command: sellprice
java.lang.ClassCastException: net.minecraft.item.crafting.Ingredient cannot be cast to net.minecraft.item.ItemStack
        at com.forgeessentials.economy.commands.CommandSellprice.calcPriceList(CommandSellprice.java:171) ~[CommandSellprice.class:?]
        at com.forgeessentials.economy.commands.CommandSellprice.parse(CommandSellprice.java:93) ~[CommandSellprice.class:?]
        at com.forgeessentials.core.commands.ParserCommandBase.func_184881_a(ParserCommandBase.java:22) ~[ParserCommandBase.class:?]
        at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:119) [bj.class:?]
        at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:91) [bj.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:960) [pa.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:939) [pa.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:37) [la.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:9) [la.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_131]
        at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_131]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:723) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_131]`

**Comment by spacebuilder2020** [comment]
Fixed with 880ac3c071913e1c739e11d3c0422cf15fff7242
